src/components/ecosystem/ecosystem-info.vue

```diff
@@ -0,0 +1,217 @@
+<script>
+import { ECOSYSTEM_DOMAIN, parseEcosystemDomain } from '~/const'
+import ipfsy from '~/utils/ipfsy'
+import { validation } from '~/mixins/validation'
+
+export default {
+  name: 'ecosystem-info',
+  mixins: [validation],
+  components: {
+    InputFileIpfs: () => import('~/components/ipfs/input-file-ipfs.vue'),
+    LoadingSpinner: () => import('~/components/common/loading-spinner.vue')
+  },
+
+  props: {
+    data: {
+      type: Object,
+      default: () => {}
+    },
+
+    isAdmin: {
+      type: Boolean,
+      default: false
+    }
+
+  },
+
+  data () {
+    return {
+      ECOSYSTEM_DOMAIN,
+      form: {
+        name: '',
+        logo: '',
+        domain: ECOSYSTEM_DOMAIN.SOCIO_ECOLOGICAL,
+        purpose: '',
+        marketplace: false,
+        ...this.data
+      },
+
+      isEditing: false,
+      isOpen: true,
+      isUploadingLogo: false
+    }
+  },
+
+  computed: {
+    hasBasicInfo () { return this.data && this.data.name && this.data.purpose }
+  },
+
+  methods: {
+    async _handleSave () {
+      try {
+        const isValid = await this.validate(this.form)
+        if (isValid) {
+          this.$emit('save', this.form)
+          this.isEditing = false
+        }
+      } catch (e) {
+        const message = e.message || e.cause.message
+        this.showNotification({ message, color: 'red' })
```

Comment:
@arsenijesavic Probably notification colour should be retrieved from colours object that contains all available colours. So here you would have something like `color: NOTIFICATION_COLOR.ERROR`